torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -949,6 +988,9 @@ def postprocessor(output):
             has_bias=has_bias,
             epilogue_creator=epilogue_creator,
             should_block_weights=block_weights,
+            name=micro_gemm.__class__.__name__,
```

Comment:
Why we need to give the name explicitly?

Reply:
By doing this, the name of microkernel is printed during max-autotune, which makes it more clearer for debugging. Thanks.